rust/processor/src/db/common/models/default_models/raw_block_metadata_transactions.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::utils::util::{compute_nanos_since_epoch, parse_timestamp, standardize_address};
+use aptos_protos::{transaction::v1::BlockMetadataTransaction, util::timestamp::Timestamp};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RawBlockMetadataTransaction {
+    pub version: i64,
+    pub block_height: i64,
+    pub id: String,
+    pub round: i64,
+    pub epoch: i64,
+    pub previous_block_votes_bitvec: String,
+    pub proposer: String,
+    pub failed_proposer_indices: String,
+    pub timestamp: chrono::NaiveDateTime,
+    pub since_unix_epoch: u64,
```

Comment:
nit: should be `ns_since_unix_epoch` (it's ns right? should have some indicator of what value is)

Reply:
done!